gitbook-docs/market/user-guides/README.md

```diff
@@ -1,24 +1,11 @@
 ---
-description: Command line guide to proof market
+description: Guides for using the Proof Market via CLI
```

Comment:
Just Proof Market, without article. The same for other mentions below

Reply:
@SK0M0R0H it seems like now we use the Proof Market almost everywhere in our documentation. @NickVolynkin should we not?